stats/opencensus/e2e_test.go

```diff
@@ -237,6 +237,23 @@ func distributionDataLatencyCount(vi *viewInformation, countWant int64, wantTags
 	return nil
 }
 
+// waitForServerCompletedRPCs waits until both Unary and Streaming metric rows
+// appear for server completed RPC's view (by checking for length of rows to be
+// 2). Returns an error if both the Unary and Streaming metric not found within
+// the passed context's timeout.
+func waitForServerCompletedRPCs(ctx context.Context, fe *fakeExporter) error {
```

Comment:
Unused function argument: you may want to remove `fe *fakeExporter` from `waitForServerCompletedRPCs` function.

Reply:
Ah, great catch. Deleted.